applications/serial_lte_modem/doc/AT_commands_intro.rst

```diff
@@ -1,7 +1,7 @@
 .. _SLM_AT_intro:
 
-AT commands
-###########
+SLM-specific AT commands
+########################
 
 The application sample uses a series of proprietary AT commands to let the nRF91 development kit operate as a Serial LTE Modem (SLM).
 
```

Comment:
Maybe we could include the link to the modem AT commands here as well?

Reply:
added :)